GET2019-E19-1219/src/Assignment1/Product.java

```diff
@@ -0,0 +1,53 @@
+package Assignment1;
+
+public class Product {
+
+	// variables
+	public int ProductQuantity;
+	public double ProductPrice;
+	public String ProductName;
+	public int ProductId;
+
+	public int getProductId() {
+		return ProductId;
+	}
+
+	public void setProductId(int productId) {
+		ProductId = productId;
+	}
+
+	public Product() {
+
+	}
+
+	public Product(int ProductId, String ProductName, double ProductPrice) {
+		this.ProductId = ProductId;
+		this.ProductName = ProductName;
+		this.ProductPrice = ProductPrice;
+	}
+
+	public int getProductQuantity() {
+		return ProductQuantity;
+	}
+
+	public void setProductQuantity(int productQuantity) {

```

Comment:
Mention comments for method. do for other methods also.

Reply:
ok sir from now on i will add comments for methods